docs/adguard-for-android/solving-problems/logcat.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: How to get Logcat log
+title: How to get system logs
 sidebar_position: 4
 ---
 
 :::info
 
-This article covers AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, [download the AdGuard app](https://adguard.com/download.html?auto=true)
+This article covers AdGuard for Android, a multifunctional ad blocker that protects your device at the system level. To see how it works, first-hand [download the AdGuard app](https://adguard.com/download.html?auto=true)
```

Comment:
To see how it works for yourself/firsthand, [download the AdGuard app](https://adguard.com/download.html?auto=true)
/